compliance/azure/compliance_score/README.md

```diff
@@ -0,0 +1,38 @@
+## Azure Regulatory Compliance Report
+
+### What it does
+
+This Policy Template will provide an overview for the various Regulatory Compliance controls and compliance scores. The policy will generate an email report.
```

Comment:
Change last sentence to ... generate an email notification or remove the whole sentence.